site/content/configuration/global-configuration/configmap-resource.md

```diff
@@ -195,10 +195,10 @@ Zone Sync enables the [ngx_stream_zone_sync_module](https://nginx.org/en/docs/st
 {{</bootstrap-table>}}
 
 {{< note >}} 
-If you previously installed OIDC or used `zone_sync` in the stream-snippets in [v4.0.1](https://github.com/nginx/kubernetes-ingress/tree/v4.0.1) or earlier, and you plan to enable the `zone-sync` ConfigMap key, please remove `zone_sync` from the old [stream-snippets](https://github.com/nginx/kubernetes-ingress/blob/v4.0.1/examples/custom-resources/oidc/nginx-config.yaml#L7).  Otherwise, NGINX Plus will encounter duplicate directive error `[emerg] 13#13: "zone_sync" directive is duplicate in /etc/nginx/nginx.conf:164`.
-Once upgraded, remove the [old headless service](https://github.com/nginx/kubernetes-ingress/blob/v4.0.1/examples/custom-resources/oidc/nginx-ingress-headless.yaml) deployed for OIDC.
+If you previously installed OIDC or used the `zone_sync` directive via `stream-snippets` in [v4.0.1](https://github.com/nginx/kubernetes-ingress/tree/v4.0.1) or earlier, and you plan to enable the [`zone-sync` ConfigMap key](/nginx-ingress-controller/configuration/global-configuration/configmap-resources/#zone-sync), the `zone_sync` directive should be removed from `stream-snippets`. 
```

Comment:
In our[ style guide](https://github.com/nginx/documentation/blob/main/templates/style-guide.md#:~:text=x%20%2D%202.x).-,via,-Do%20not%20use), we avoid "latin" terms like "via"

```suggestion
If you previously installed OIDC or used the `zone_sync` directive with `stream-snippets` in [v4.0.1](https://github.com/nginx/kubernetes-ingress/tree/v4.0.1) or earlier, and you plan to enable the [`zone-sync` ConfigMap key](/nginx-ingress-controller/configuration/global-configuration/configmap-resources/#zone-sync), the `zone_sync` directive should be removed from `stream-snippets`. 
```